models/sessions.go

```diff
@@ -0,0 +1,66 @@
+package models
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/gobuffalo/pop/v5"
+	"github.com/gofrs/uuid"
+	"github.com/netlify/gotrue/storage"
+	"github.com/pkg/errors"
+)
+
+type Session struct {
+	ID        uuid.UUID `json:"-" db:"id"`
+	UserID    uuid.UUID `json:"user_id" db:"user_id"`
+	CreatedAt time.Time `json:"created_at" db:"created_at"`
+	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
+}
+
+func (Session) TableName() string {
+	tableName := "sessions"
+	return tableName
+}
+
+func NewSession(user *User) (*Session, error) {
+	id, err := uuid.NewV4()
+	if err != nil {
+		return nil, errors.Wrap(err, "Error generating unique session id")
+	}
+	session := &Session{
+		ID:     id,
+		UserID: user.ID,
+	}
+	return session, nil
+}
+
+func CreateSession(tx *storage.Connection, user *User) (*Session, error) {
+	session, err := NewSession(user)
+	if err != nil {
+		return nil, err
+	}
+	if err := tx.Create(session); err != nil {
+		return nil, errors.Wrap(err, "error creating session")
+	}
+	return session, nil
+}
+
+func FindSessionById(tx *storage.Connection, id uuid.UUID) (*Session, error) {
```

Comment:
Can we use a capital `ID` I think that's what is generally used throughout the code.